genmod/utils/get_features.py

```diff
@@ -1,68 +1,98 @@
-from __future__ import (print_function)
+from __future__ import print_function
 
 import logging
 
-from genmod.utils import INTERESTING_SO_TERMS, EXONIC_SO_TERMS
+INTERESTING_SO_TERMS = {
+    "transcript_ablation",
+    "splice_donor_variant",
+    "splice_acceptor_variant",
+    "stop_gained",
+    "start_lost",
+    "frameshift_variant",
+    "stop_lost",
+    "initiator_codon_variant",
```

Comment:
Another list of SO terms 😱 
I don't think that this one for example exists anymore. Don't know what to do with it but let it be as is for now

Reply:
Yes: 😱. I noted with Felix LRS annotation issues that `bcftools +split-vep` had an ordered one of its own as well, for when someone wants to retain only worst consequence transcript annotations.